logfire/_internal/metrics.py

```diff
@@ -25,8 +25,10 @@
 try:
     # This only exists in opentelemetry-sdk>=1.23.0
     from opentelemetry.metrics import _Gauge as Gauge
+
+    GAUGE_IMPORTED = True
 except ImportError:  # pragma: no cover
-    Gauge = None
+    GAUGE_IMPORTED = False  # type: ignore
```

Comment:
I don't see why this is better than `Gauge = None`, and it requires 2 extra type-ignores.

Reply:
It actually avoids the type ignore on the type hint of the `metric_gauge` method - but I just had an idea on how to avoid one more.